builders/base/README.md

```diff
@@ -0,0 +1,86 @@
+## Oryx Builder
+
+### Steps for releasing new builder
+
+#### Prerequisites
+
+You must have access to the `oryxprodmcr` ACR instance in the Oryx production subscription account and have logged in
+locally to `docker` with this ACR instance's credentials.
+
+#### Build CLI image
+
+_Note_: The CLI Builder image used has its OS packages pre-baked, since the builder **CAN NOT** run as a root user and
```

Comment:
Nit: it might be worth completing the sentence to say that we cannot do that because the builder will be running in an unprivileged environment. I don't believe it's a fundamental limitation of Buildpack or something, more just the place where we'll run.

Reply:
So actually the CNB docs state that the CNB_USER can not be a root user, https://buildpacks.io/docs/operator-guide/create-a-stack/#specification. I've added a link to that doc to the README

`
The CNB_USER_ID is the UID of the user as which the detect and build steps are run. The given user MUST NOT be a root user and have it’s home directly writeable. CNB_GROUP_ID is the primary GID of the above user.
`